resources/lang/vi/validation.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Validation Language Lines
+|--------------------------------------------------------------------------
+|
+| The following language lines contain the default error messages used by
+| the validator class. Some of these rules have multiple versions such
+| as the size rules. Feel free to tweak each of these messages here.
+|
+*/
+
+return [
+    'accepted'             => 'Trường :attribute phải được chấp nhận.',
```

Comment:
```suggestion
    'accepted' => 'Trường :attribute phải được chấp nhận.',
```
Sửa các chỗ tương tự nha.